src/processIssueComment.js

```diff
@@ -42,18 +42,24 @@ async function processAddContributor({
         originalSha: optionsConfig.getOriginalSha(),
     }
 
-    const pullRequestURL = await repository.createPullRequestFromFiles({
-        title: `docs: add ${who} as a contributor`,
-        body: `Adds @${who} as a contributor for ${contributions.join(
-            ', ',
-        )}.\n\nThis was requested by ${commentReply.replyingToWho()} [in this comment](${commentReply.replyingToWhere()})`,
-        filesByPath: filesByPathToUpdate,
-        branchName: `all-contributors/add-${who}`,
-    })
+    if (contributions.length) {
```

Comment:
Could we use a guard approach instead? https://link.medium.com/klVQCctMvT

Reply:
Sure.